UNL-CSE-AMBASSADORS/VITA
# Cancel Confirmation Dialog for Volunteers

Added in a cancel confirmation dialog for volunteers when they cancel an appointment. This code is similar to what is happening for the self-service client rescheduling.

**TESTING**
1. Sign in as a preparer
2. Go to appt mgmt page
3. Select any appointment
4. Scroll down and hit the cancel button
    * It should open up a dialog prompting the user to confirm they want to cancel the appointment
        * Clicking outside the dialog should close it and not cancel the appt
        * Clicking the X at the top right of the dialog should close it and not cancel the appt
        * Clicking the "No" button should close the dialog and not cancel the appt
5. Hit "Yes"
    * The appointment should now show as cancelled, the dialog should automatically close, and a success notification should appear
